### DISCUSSION TITLE
Redesigning SendGrid Sms experience with Paste

### DISCUSSION THREAD

**Discussion by AlejandraOsorioGarzon** [twilio-labs/paste#4078] [Office hours] [answered]
### Background

Hey! i'm updating the components in the sms flow in SendGrid which is going to be GA in Q4. I want to have a session with the design system team to check if everything is ok in the designs and get feedback

### Feedback

Check that all components are being used correctly and doing kind of a QA.

### Stage of design

High fidelity

https://www.figma.com/design/YfLCdDtOHqm0mNomiZ8vDn/SMS-in-MC-improvements?node-id=190-71127&t=sOXa4UJUGCs8l3zT-1

_No response_

### Are you looking for visual design support?

- [x] Yes

**Comment by serifluous** [comment]
Hey @AlejandraOsorioGarzon ! Can you share a Figma file so we can prepare? Feel free to link it here or in Slack (#help-design-system).

**Comment by roxanagomez** [comment]
Hi @AlejandraOsorioGarzon thank you for sharing your Figma file, we left some [initial recommendations](https://www.figma.com/design/YfLCdDtOHqm0mNomiZ8vDn?node-id=51-6894#961038028) on you file to have some points of conversation for Office Hours, let us know if you have any other areas of that document you would like us to take a look at.

**Comment by krisantrobus** [comment, marked as answer]
Hey @AlejandraOsorioGarzon  - just wanted to share the office hours recording with you in case you'd like to revisit any of our conversations. [Here's the video](https://urldefense.com/v3/__https://twilio.zoom.us/rec/share/-QLYjny-_IqabiWVJgGDLUB_UZAK2RM6NR17mEeL2jdZEUMuXopN3A9xwiwDyHPi.st6XSsn5VIV7LJ1S?pwd=sCQOMFMXpVNq7d4TWqIL4eY9djFRY0Ra__;!!NCc8flgU!eQMUiEWpcwRVTNDakUF3-TNK0P5e8dChvlZaLaWntRN7cKpnFYnV61q12ENCTTgmidkdq0mhkV6VZJY6kg$). Thanks again for coming to office hours!
